app/src/main/java/com/cris/nvh/framgiaproject/mediaplayer/MediaPlayerManager.java

```diff
@@ -9,20 +9,27 @@
 
 import java.io.IOException;
 import java.util.List;
+import java.util.Random;
 
 public class MediaPlayerManager extends MediaPlayerSetting implements PlayMusic,
 		MediaPlayer.OnCompletionListener, MediaPlayer.OnPreparedListener {
 
+	private static final int NUMBER_1 = 1;
```

Comment:
```suggestion
	private static final int CHANGE_POSITION = 1;
```e có thể đặt tên có ý nghĩa hơn.